lib/jackal-slack/notification.rb

```diff
@@ -14,7 +14,9 @@ def setup(*_)
       # @return [Truthy, Falsey]
       def valid?(message)
         super do |payload|
-          payload.get(:data, :slack)
```

Comment:
True, but we can just do the get directly:

```ruby
super do |payload|
  config[:webhook_url] && payload.get(:data, :slack, :messages).is_a?(Array)
end
```
emptiness can be checked, but it won't really matter since empty will just result in it processing with no output.